### ISSUE TITLE
Parameter name changes / removal

### ISSUE THREAD

**Issue by nefrathenrici** [CliMA/CLIMAParameters.jl#156]
Please list any breaking parameter changes here - they will be bundled together and added before a new minor version is released.

**Comment by nefrathenrici** [comment]
```julia
        B1994_auctoconversion_coeff_C,
        B1994_auctoconversion_coeff_a,
        B1994_auctoconversion_coeff_b,
        B1994_auctoconversion_coeff_c,
        B1994_auctoconversion_coeff_N_0,
        B1994_auctoconversion_coeff_d_low,
        B1994_auctoconversion_coeff_d_high,
        threshold_smooth_transition_steepness,
```
Should be renamed from `auctoconversion` -> `autoconversion`

**Comment by trontrytel** [comment]
That would be great, Here is a draft PR: https://github.com/CliMA/CLIMAParameters.jl/pull/168

**Comment by nefrathenrici** [comment]
Should `snow_cross_section_size_relation_coefficient_dela` and `rain_cross_section_size_relation_coefficient_dela` be renamed to `rain_cross_section_size_relation_coefficient_deltaa`? A similarly named parameter is `rain_cross_section_size_relation_coefficient_chia`.

**Comment by nefrathenrici** [comment]
The `Chen2022_table_B3` set of parameters should be turned into array parameters.

**Comment by trontrytel** [comment]
> Should `snow_cross_section_size_relation_coefficient_dela` and `rain_cross_section_size_relation_coefficient_dela` be renamed to `rain_cross_section_size_relation_coefficient_deltaa`? A similarly named parameter is `rain_cross_section_size_relation_coefficient_chia`.

Lets keep with the naming convention there. It's not the prettiest but it's not a typo either.

**Comment by nefrathenrici** [comment]
Closed by #168, thank you @trontrytel !
